pages/common/coffee.md

```diff
@@ -0,0 +1,23 @@
+# coffee
+
+> Executes CoffeeScript scripts or compiles them into JavaScript.
+
+- Run script:
+
+`coffee {{filename}}.coffee`
+
+- Compile to JavaScript and save file:
+
+`coffee --compile {{filename}}.coffee`
+
+- Compile to JavaScript and save as file:
```

Comment:
Perhaps ...`and output to a specific file` would read better?

Also, could `{{filename}}` be `{{path/to/filename.js}}` perhaps? That would make it more consistent with the existing tldr-pages we have here.

Reply:
Ah, I see